### DISCUSSION TITLE
Slots content disappear

### DISCUSSION THREAD

**Discussion by nikitabelotelov** [lit/lit#2557] [Q&A] [answered]
I'm using the lit element for writing web components. But I run into a strange bug with my select list. I've tried to simplify the code of it and put it to this demo.
https://codesandbox.io/s/litelement-forked-ol9d7?file=/src/index.js
You can click on the text, then select one item, then click again and select another. And the layout will disappear.
Seems like it happens because of this code
`slot=${this.open ? "hint" : ""}`
because if you replace it with
`slot=hint`
then you won't be able to reproduce the bug.
And another strange thing is that if you uncheck/check any styles in the inspector(for example, uncheck font-family style on body) the layout will become visible again. This and also the fact that it works well in Safari make me think, that the bug is in Chrome itself. But I decided to create an issue here at first. So should I create an issue for Chrome or it might be a problem in lit-element lib?

**Comment by sorvell** [comment, marked as answer]
I think this is almost certainly a Chrome bug. It would be great if you could file a Chrome issue.

Any time twiddling a style on/off like this causes rendering to change, there's a likely browser bug.
